src/functions/blogMethods/custom-mdx/add-frontmatter.js

```diff
@@ -6,13 +8,14 @@ const addFrontmatter = ({
   firstPublishedAt,
   authorName,
 }) => `---
-title: ${title}
```

Comment:
Can you add a test to `add-frontmatter.test.js` where the title contains a colon please